docs/configuration.md

```diff
@@ -120,6 +120,29 @@ jobs:
   trigger: commit
 ```
 
+**copr\_build**
+
+Gather data from new pull_request or release and use trigger new copr build with it.
+
+Supported triggers: **pull_request**, **release**.
+
+
+**Example**
+
+```yaml
+jobs:
+- job: copr_build
+  trigger: pull_request
+  metadata:
+    targets:
+      - fedora-rawhide-x86_64
+      - fedora-30-x86_64
+    # (Optional) Defaults to 'packit'
+    owner: some_copr_project_owner
```

Comment:
for service: I don't think this is useful since our packit identity won't be able to build in that namespace, right?

so, I would remove it from user docs and added it to contribution.md for packit devs